apis/rds/v1alpha1/zz_db_cluster.go

```diff
@@ -0,0 +1,475 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// DBClusterParameters defines the desired state of DBCluster
+type DBClusterParameters struct {
+	// Region is which region the DBCluster will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// A list of Availability Zones (AZs) where instances in the DB cluster can
+	// be created. For information on AWS Regions and Availability Zones, see Choosing
+	// the Regions and Availability Zones (https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.RegionsAndAvailabilityZones.html)
+	// in the Amazon Aurora User Guide.
+	AvailabilityZones []*string `json:"availabilityZones,omitempty"`
+	// The target backtrack window, in seconds. To disable backtracking, set this
+	// value to 0.
+	//
+	// Currently, Backtrack is only supported for Aurora MySQL DB clusters.
+	//
+	// Default: 0
+	//
+	// Constraints:
+	//
+	//    * If specified, this value must be set to a number from 0 to 259,200 (72
+	//    hours).
+	BacktrackWindow *int64 `json:"backtrackWindow,omitempty"`
+	// The number of days for which automated backups are retained.
+	//
+	// Default: 1
+	//
+	// Constraints:
+	//
+	//    * Must be a value from 1 to 35
+	BackupRetentionPeriod *int64 `json:"backupRetentionPeriod,omitempty"`
+	// A value that indicates that the DB cluster should be associated with the
+	// specified CharacterSet.
+	CharacterSetName *string `json:"characterSetName,omitempty"`
+	// A value that indicates whether to copy all tags from the DB cluster to snapshots
+	// of the DB cluster. The default is not to copy them.
+	CopyTagsToSnapshot *bool `json:"copyTagsToSnapshot,omitempty"`
+	// The name of the DB cluster parameter group to associate with this DB cluster.
+	// If you do not specify a value, then the default DB cluster parameter group
+	// for the specified DB engine and version is used.
+	//
+	// Constraints:
+	//
+	//    * If supplied, must match the name of an existing DB cluster parameter
+	//    group.
+	DBClusterParameterGroupName *string `json:"dbClusterParameterGroupName,omitempty"`
+	// A DB subnet group to associate with this DB cluster.
+	//
+	// Constraints: Must match the name of an existing DBSubnetGroup. Must not be
+	// default.
+	//
+	// Example: mySubnetgroup
+	DBSubnetGroupName *string `json:"dbSubnetGroupName,omitempty"`
```

Comment:
This can be a reference to `DBSubnetGroup` in current `database/v1beta1` package.